lib/Dialect/Seq/SeqPasses.cpp

```diff
@@ -43,15 +43,21 @@ struct CompRegLower : public OpConversionPattern<CompRegOp> {
     Location loc = reg.getLoc();
 
     StringAttr name = reg.nameAttr();
-    if (!name)
+    if (reg.name().empty())
       name = rewriter.getStringAttr("_compreg");
     auto svReg =
         rewriter.create<sv::RegOp>(loc, reg.getResult().getType(), name);
-    DictionaryAttr regAttrs = reg->getAttrDictionary();
-    if (!regAttrs.empty())
-      svReg->setAttrs(regAttrs);
-    auto regVal = rewriter.create<sv::ReadInOutOp>(loc, svReg);
 
+    // Carry forward all the attributes from CompRegOp which don't conflict.
```

Comment:
Please do not pull forward random attributes from one op onto the other.  Please only specify the attributes that `sv.reg` expects.

Reply:
Should I only pull forward the discardable attributes with a dialect name prefix?